src/Styleguide/Components/ArtistCard.tsx

```diff
@@ -6,13 +6,10 @@ import { Flex } from "Styleguide/Elements/Flex"
 import { Spacer } from "Styleguide/Elements/Spacer"
 import { Responsive } from "Styleguide/Utils/Responsive"
 
-export interface ArtistCardProps {
-  src: string
-  headline: string
-  subHeadline: string
-}
+import { ArtistCard_artist } from "__generated__/ArtistCard_artist.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
 
-export class ArtistCard extends React.Component<ArtistCardProps> {
+export class ArtistCard extends React.Component<ArtistCard_artist> {
```

Comment:
Oh nice, just pass the generated fragment right in

Reply:
👍

Reply:
This can’t be right, because the `ArtistCard_artist` describes the `artist` prop. i.e. it should be:

```ts
interface Props {
  artist: ArtistCard_artist
}

export class ArtistCard extends React.Component<Props> {
  // ...
}
```

Reply:
This PR works though...

Reply:
I mean everything renders fine. I'm confused what the issue is, since the Relay compiler isn't emitting any messages (although removing `@relay(mask: false)` breaks it).